Gems/WhiteBox/Code/Source/Components/WhiteBoxColliderComponent.h

```diff
@@ -18,7 +18,10 @@
 
 namespace WhiteBox
 {
-    //! Runtime White Box Collider.
+    //! Component that provides a White box Collider and associated Rigid Body.
+    //! 
```

Comment:
I would suggest just getting rid of this line.

Reply:
Sorry, which line would you remove? I've tried to make it so both Heightfield and Whitebox comments are similar because they are the same case.

Reply:
I meant the blank line in the comment.